app/services/proceeding_types_service.rb

```diff
@@ -0,0 +1,21 @@
+class ProceedingTypesService
+  def initialize(legal_aid_application)
+    @legal_aid_application = legal_aid_application
+  end
+
+  def add(proceeding_type_id: nil, scope_limitation: nil)
+    return false unless proceeding_type_id && scope_limitation
+
+    ActiveRecord::Base.transaction do
+      @legal_aid_application.proceeding_types << proceeding_type(proceeding_type_id)
+      AddScopeLimitationService.call(@legal_aid_application, scope_limitation)
+    end
+    true
```

Comment:
I don't think it's necessary to return true here as line 9 returns an activerecord object anyway which is truthy